MEAnalysis/python/JetAnalyzer.py

```diff
@@ -33,9 +33,15 @@ def __init__(self, cfg_ana, cfg_comp, looperName):
         super(JetAnalyzer, self).__init__(cfg_ana, cfg_comp, looperName)
         self.conf = cfg_ana._conf
 
+    def beginLoop(self, setup):
+        super(JetAnalyzer, self).beginLoop(setup)
+        self.inputCounter = ROOT.TH1F("JetAnalyzer_Count","Count",1,0,2)
+        self.inputCounterPosWeight = ROOT.TH1F("JetAnalyzer_CountPosWeight","Count genWeight>0",1,0,2)
+        self.inputCounterNegWeight = ROOT.TH1F("JetAnalyzer_CountNegWeight","Count genWeight<0",1,0,2)
```

Comment:
@d-salerno Actually, counting neg/pos weights here is not entirely correct: the vhbb code already does some pre-filtering of events, which may bias this ratio. we currently have neg/pos weight counting already in the vhbb code via histograms, so I will remove this part.